### PR TITLE
Install real chkconfig

### PR DESCRIPTION
**Pull request by jayvdb**
chkconfig is a dependency of oracle-xe installer, so it would be good to use the actual package, and not create an empty shell script.
It was whitelisted as an apt package (https://github.com/travis-ci/apt-package-whitelist/issues/2465), and is used by https://github.com/scop/bash-completion

### CODE REVIEW

**Comment by jayvdb** [comment]
As you may be able to guess, this is a refined chunk from the container patch.

File: install.sh [hunk 1]
```diff
@@ -7,14 +7,21 @@ ORACLE_RPM="$(basename $ORACLE_FILE .zip)"
...
-  ( sudo apt-get -qq update && sudo apt-get --no-install-recommends -qq install bc libaio1 rpm unzip )
+DEPS="bc libaio1 rpm unzip"
+if [ "$_system_version" = "12.04" ]; then
+   DEPS="$DEPS chkconfig"
+fi
+dpkg -s $DEPS > /dev/null 2>&1 ||
+  ( sudo apt-get -qq update && sudo apt-get --no-install-recommends -qq install $DEPS )
+
+# chkconfig expects /sbin/insserv to exist
+sudo ln -s /usr/lib/insserv/insserv /sbin/insserv
```
Comment (jayvdb) [inline install.sh:18]: This sudo will need to be revised in the container patch. Maybe I can container-proof it now..

Comment (cbandy) [inline install.sh:18]: So installing the `chkconfig` package results in a broken `chkconfig` command?

File: install.sh [hunk 2]
```diff
@@ -7,14 +7,21 @@ ORACLE_RPM="$(basename $ORACLE_FILE .zip)"
...
 
 df -B1 /dev/shm | awk 'END { if ($1 != "shmfs" && $1 != "tmpfs" || $2 < 2147483648) exit 1 }' ||
   ( sudo rm -r /dev/shm && sudo mkdir /dev/shm && sudo mount -t tmpfs shmfs -o size=2G /dev/shm )
 
 test -f /sbin/chkconfig ||
-  ( echo '#!/bin/sh' | sudo tee /sbin/chkconfig > /dev/null && sudo chmod u+x /sbin/chkconfig )
+  ( echo '#!/bin/sh' | sudo tee /sbin/chkconfig > /dev/null && sudo chmod 755 /sbin/chkconfig )
```
Comment (cbandy) [inline install.sh:24]: Why this change?

File: install.sh (hunk 1 above)
Comment (jayvdb) [inline install.sh:18]: Essentially yes, but it is fixable.

File: install.sh (hunk 2 above)
Comment (jayvdb) [inline install.sh:24]: 755 is the permissions of the packaged version

**Comment by jayvdb** [comment]
An alternative is to restore the original `chkconfig` if it exists, or just ignore chkconfig is missing and ignore the stderr message produced because it is missing.

File: install.sh (hunk 1 above)
Comment (jayvdb) [inline install.sh:18]: I couldnt find a neat way to add this, in a way that doesnt need to be rewritten after the container support is merged.  As a result, IMO, container support should be merged first, and we re-evaluate this chkconfig problem after that.

**Comment by cbandy** [comment]
> chkconfig is a whitelisted package, so it shouldnt be destroyed by travis-oracle.

We create `/sbin/chkconfig` when it does not exist. How does your project use `chkconfig`, and what problem does this fix for you?

**Comment by jayvdb** [comment]
> We create /sbin/chkconfig when it does not exist.

True, and since it doesnt clobber an existing file, such as if it was already installed, this PR has very little value.
